src/vm/stublink.cpp

```diff
@@ -1327,7 +1327,11 @@ bool StubLinker::EmitUnwindInfo(Stub* pStub, int globalsize)
         COMPlusThrowOM();
     }
 
+#ifndef FEATURE_PAL
```

Comment:
Rather unsure of this change, we don't have AllocationBase with PAL, so this was my best educated guess.